doc/TRY.md

```diff
@@ -144,24 +96,20 @@ or the simulator of any of them (see the links at the top of this document).
 
 Click on next. If you want to try restoring from wallet backup later on, make sure to backup the
 mnemonic as well as the descriptor in the next two screens. Otherwise just make them happy by
-ticking the boxes. If you are using a signing device simulator you'll have a step for registering
+ticking the boxes. If you are using a signing device or its simulator you'll have a step for registering
 the descriptor on it.
 
 You can then decide whether you would like to manage `bitcoind` yourself or let Liana configure
 and start/stop it while the GUI is being used:
-- If you choose to manage `bitcoind` yourself, the next step will be to configure the connection.
-The default should work for what we did in this guide.
-- If you choose to let Liana manage `bitcoind`, the next step will search for a `bitcoind`
-executable on your computer and start it, using `<Liana data directory>/bitcoind_datadir` as the
-data directory and creating a `bitcoin.conf` file therein.
+For the purpose of this guide, we will use the simpler option: to let Liana download and managed Bitcoin Core for us. It will get the software on Bitcoincore.org and configure it in Pruned mode with about 20GB of disk usage.
```

Comment:
```suggestion
For the purpose of this guide, we will use the simpler option: to let Liana download and managed Bitcoin Core for us. It will get the software on [bitcoincore.org](https://bitcoincore.org/) and configure it in pruned mode with about 20GB of disk usage.
```